packages/assets-controllers/src/TokensController.test.ts

```diff
@@ -852,6 +851,54 @@ describe('TokensController', () => {
       ).toStrictEqual(dummyTokens);
     });
 
+    it('should nest allTokens under chain ID and selected address when detectedTokens provided is an empty list', async () => {
+      tokensController.configure({
+        selectedAddress: dummySelectedAddress,
+        chainId: NetworksChainId.mainnet,
+      });
+      await tokensController.addTokens(dummyTokens);
+      const processedTokens = tokensController._getNewAllTokensState({
```

Comment:
Considering that this is a method that starts with a underscore, one would assume that it is effectively private. I did a quick search and it doesn't appear that either the extension or mobile app call this method directly. What do you think about checking `tokensController.state` directly?

Reply:
I was just mimicking one of the previous tests for this scenario,  maybe we can send an empty list in `addTokens` and check for state change in `tokensController.state` as well.
And the same for the next two test as well.